src/components/countdown/CountdownSettings.vue

```diff
@@ -17,10 +23,29 @@ import BaseSettings from '../presentation/BaseSettings.vue'
 
 export default {
   extends: BaseSettings,
+  data () {
+    return {
+      klokopties: [
+        {
+          label: 'Aftelklok',
+          value: 0,
+          omschrijving: 'Aftellen tot (weergave bij tijd in de toekomst op dezelfde dag)'
+        },
+        {
+          label: 'Klok',
+          value: 1,
+          omschrijving: 'Klok tot (weergave bij tijd in de toekomst op dezelfde dag)'
+        }
+      ]
+    }
+  },
   created () {
     if (!this.settings.time) {
       this.settings.time = this.$store.service.time
     }
+    if (this.settings.klok == null) {
```

Comment:
Je wil hier checken of `settings.klok` bestaat denk ik? Dan moet je op `undefined` checken:
```suggestion
    if (this.settings.klok === undefined) {
```

Reply:
Klopt dat probeerde ik te doen anders wordt bij de options keuzes niet iets geselecteerd. En kan hij daardoor ook de waarde voor in tekstvak niet uitlezen.